### PR TITLE
Update to 0.6.6

### PR DESCRIPTION
**Pull request by ericdill**
attn @parente

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ source:
 
 build:
   number: 0
+  skip: True  # [py2k]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Will need to re-render if you are adding this.

Does it not support Python 2 any more?

Comment (ericdill) [inline recipe/meta.yaml]: Version 0.6.5 doesn't actually support Python 2 either. It uses a py3 only feature of tempfile: https://github.com/maxpoint/conda-mirror/blob/master/conda_mirror/conda_mirror.py#L510

Comment (jakirkham) [inline recipe/meta.yaml]: Ah, oops, sorry. 😳 I can pull the Python 2 package if you would like.

Comment (jakirkham) [inline recipe/meta.yaml]: Also we need the re-render with 2.2.1+ (2.2.2 would be best) as it fixes a bug on new AppVeyor images.

Comment (ericdill) [inline recipe/meta.yaml]: No worries. I should have caught that when you submitted the PR to staged-recipes. I would be grateful if you could pull the py2 packages. Sorry for the extra work for you

Comment (jakirkham) [inline recipe/meta.yaml]: Not a problem. Have moved them under the `broken` label.

**Comment by ericdill** [comment]
CI services got a little wonky on the last commit because I accidentally pushed the branch directly to the repo at conda-forge/conda-mirror-feedstock. I immediately deleted it which is why there is an extra "travis-ci/push" and an "appveyor/branch" check. Both of those can safely be ignored if they have failed

File: recipe/meta.yaml (hunk 1 above)
Comment (ericdill) [inline recipe/meta.yaml]: Great. Thanks @jakirkham !

**Pull request merged**